docs/expo.md

```diff
@@ -0,0 +1,163 @@
+npx expo run:ios
+```
+
+## Install Adobe Mobile SDKs
+
+Follow the [Installation Guide](../README.md#Installation) to install Adobe SDKs after the android and ios directories are generated.
+
+## Initialize Adobe Mobile SDKs
+Initializing the SDK should be done in native code inside your `AppDelegate` (iOS) and `MainApplication` (Android). The following code snippets demonstrate how to install and register the AEP Mobile Core and Edge Network extensions. Documentation on how to initialize each extension can be found in _./packages/{extension}/README.md_.
+
+##### **iOS**
+
+```objective-c
+//AppDelegate.h
+@import AEPCore;
+@import AEPServices;
+@import AEPLifecycle;
+@import AEPSignal;
+@import AEPEdge;
+@import AEPEdgeIdentity;
+@import AEPEdgeConsent;
+...
+```
+
+```objective-c
+//AppDelegate.m
+...
+@implementation AppDelegate
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  self.moduleName = @"AEPSampleApp";
+  self.initialProps = @{};
+  
+  [AEPMobileCore setLogLevel:AEPLogLevelTrace];
+  [AEPMobileCore configureWithAppId:@"YOUR-APP-ID"];
+  const UIApplicationState appState = application.applicationState;
+  [AEPMobileCore registerExtensions:@[
+      AEPMobileLifecycle.class, AEPMobileIdentity.class,
+      AEPMobileEdgeIdentity.class, AEPMobileEdge.class,
+      AEPMobileEdgeConsent.class, AEPMobileEdgeBridge.class
+    ]
+  completion:^{
+    if (appState != UIApplicationStateBackground) {
+      [AEPMobileCore lifecycleStart:nil];
+      }
+      }];
+
+  return [super application:application didFinishLaunchingWithOptions:launchOptions];
+}
+
+@end
+
+```
+
+> To enable the Lifecycle metrics, [implement the Lifecycle APIs](./packages/core/README.md#lifecycle)
+
+> Hint : While running iOS application after Adobe Experience Platform SDK installation. If you have build error that states:
+> "ld: warning: Could not find or use auto-linked library 'swiftCoreFoundation'"
+> This is because Adobe Experience Platform SDK now requires the app uses swift interfaces. Add a dummy .swift file to your project to embed the swift standard libs. See the SampleApp presented in this repo for example.
+
+##### **Android:** _(Kotlin)_
+
+```kotlin
+// MainApplication.kt
+import com.adobe.marketing.mobile.AdobeCallback;
+import com.adobe.marketing.mobile.Extension;
+import com.adobe.marketing.mobile.LoggingMode;
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Lifecycle;
+import com.adobe.marketing.mobile.Signal;
+import com.adobe.marketing.mobile.Edge;
+import com.adobe.marketing.mobile.edge.consent.Consent;
+```
+
+```kotlin
+class MainApplication : Application(), ReactApplication {
+  ...
+  override fun onCreate() {
+    super.onCreate()
+    ...
+
+    MobileCore.setApplication(this)
+    MobileCore.setLogLevel(LoggingMode.VERBOSE)
+    MobileCore.configureWithAppID("YOUR-APP-ID")
+    val extensions: List<Class<out Extension?>> = Arrays.asList(
```

Comment:
Remove lifecycle and signal

Reply:
Kotlin sample:
Refer to sample her for removing lifecycle
https://developer.adobe.com/client-sdks/edge/edge-network/